### PR TITLE
Issue #9824: update ParameterNameCheckTest.java to use unique input file in each test method

### PR DESCRIPTION
**Pull request by SGanguly1999**
Closes: #9824

### CODE REVIEW

**Comment by SGanguly1999** [comment]
@romani please review this PR when you are free

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheckTest.java [hunk 1]
```diff
@@ -65,11 +65,11 @@ public void testSpecified()
...
         final String[] expected = {
-            "71:19: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat1", pattern),
-            "71:34: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat2", pattern),
-            "72:25: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat3", pattern),
+            "77:19: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat1", pattern),
+            "77:34: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat2", pattern),
+            "78:25: " + getCheckMessage(MSG_INVALID_PATTERN, "badFormat3", pattern),
         };
-        verify(checkConfig, getPath("InputParameterName.java"), expected);
+        verify(checkConfig, getPath("InputParameterName_1.java"), expected);
```
Comment (timurt) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheckTest.java:72]: Lets avoid `_` character as it does not satisfy java class naming conventions, you can use `InputParameterName1` or `InputParameterNameOne` or name it according to the test case `InputParameterNameSpecified`

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java [hunk 2]
```diff
@@ -0,0 +1,231 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: Feb-2001
+// Ignore violation
+////////////////////////////////////////////////////////////////////////////////
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java:5]: could you please remove this header comment, do it for all inputs in this PR

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+ */
+
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java:21]: also remove this comment

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheckTest.java (hunk 1 above)
Comment (SGanguly1999) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheckTest.java]: @timurt done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java (hunk 2 above)
Comment (SGanguly1999) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java (hunk 3 above)
Comment (SGanguly1999) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/parametername/InputParameterName_1.java]: done

**Review by romani — APPROVED** [review]

**Pull request merged**